package.json

```diff
@@ -40,7 +40,7 @@
     "build": "tsc",
     "postbuild": "npm run copy:schematics",
     "copy:schematics": "cpy ./schematics/**/*.json ../dist --cwd=src --parents",
-    "lint:all": "eslint ./src/**/*.ts ./tests/**/*.ts --fix",
+    "lint:all": "eslint ./scripts/**/*.ts ./src/**/*.ts ./tests/**/*.ts --fix",
```

Comment:
I've also enabled this, LMK if you think it's a good ideia...

Reply:
Yea, that's fine 👍